gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -564,6 +591,18 @@ def write_initial_zarr(self):
         if isinstance(self.store, IPLD):
             self.dataset_hash = str(mapper.freeze())
 
+    def set_key_dims(self):
+        """
+        Convenience method to set the standard and time dimensions based on whether a dataset is a forecast or not
+        The self.forecast instance variable is set in the `init` of a dataset and defaults to False.
+        """
+        if not self.forecast:
+            self.standard_dims = ["time", "latitude", "longitude"]
+            self.time_dim = "time"
+        else:
+            self.standard_dims = ["forecast_reference_time", "constant_offset", "latitude", "longitude"]
```

Comment:
Could we change `"constant_offset"` to something that references time? Like "time_offset" maybe?

Reply:
`forecast_offset`?